lib/burials/monitor.rb

```diff
@@ -7,6 +7,8 @@ module Burials
   # Monitor functions for Rails logging and StatsD
   #
   class Monitor < ::ZeroSilentFailures::Monitor
+    include Logging::Monitor
```

Comment:
will this cause issues since there would now be 2 constructors? and also a double include/inherit

Reply:
Yeah I'll remove the inheritance